mqtt-cs/src/main/java/org/apache/rocketmq/mqtt/cs/session/infly/MqttMsgId.java

```diff
@@ -59,22 +59,17 @@ public int nextId(String clientId) {
         MsgIdEntry msgIdEntry = hashMsgID(clientId);
         synchronized (msgIdEntry) {
             int startingMessageId = msgIdEntry.nextMsgId;
-            int loopCount = 0;
-            int maxLoopCount = 1;
             do {
                 msgIdEntry.nextMsgId++;
                 if (msgIdEntry.nextMsgId > MAX_MSG_ID) {
                     msgIdEntry.nextMsgId = MIN_MSG_ID;
                 }
                 if (msgIdEntry.nextMsgId == startingMessageId) {
-                    loopCount++;
-                    if (loopCount >= maxLoopCount) {
```

Comment:
i think this is not necessary, the if judgment will absolutely happen when the 'loopCount' plus

Reply:
Yes, it may be the case that maxLoopCount>1 was considered before

Reply:
in the old code, the var maxLoopCount=2, but it may be cause the getNextId become slow ,then hang the thread

Reply:
Under most cases, the method 'releaseId' which release 'inUseMsgIds' is enough, unless lots of message accumulation happens. Once the extreme case occurs, i think it makes no difference or no sense to run the loop once or twice.

so may I remove the loop judgment and keep this change? @pingww @tianliuliu

Reply:
> Under most cases, the method 'releaseId' which release 'inUseMsgIds' is enough, unless lots of message accumulation happens. Once the extreme case occurs, i think it makes no difference/sense to run the loop once or twice.
> 
> so may I remove the loop judgment and keep this change? @pingww @tianliuliu

you are right, . the cause is exactly it.